08.ls_object/short_formatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(files)
   def output_file
     max_length = @files.max_by(&:length).length
 
-    divide_files_by_column(@files, COLUMN).each do |files|
+    split_files_by_column(@files, COLUMN).each do |files|
```

Comment:
```suggestion
    split_files_by_column.each do |files|
```

同じクラスで定義している定数＆インスタンス変数なので引数で渡す必要はないですね

もしくは、`@files`は`def output_file(files)`のようにメソッドの引数にするのも一手です。

オブジェクト指向的にどっちがベターなのか検討してみてください。
（LongFormatterも同様に）

Reply:
仰る通り、不必要にメソッドに引数を渡していました。
引数を渡さない処理に変更しました。